internal/codespaces/api/api.go

```diff
@@ -183,6 +183,7 @@ type Codespace struct {
 	VSCSTarget                     string              `json:"vscs_target"`
 	PendingOperation               bool                `json:"pending_operation"`
 	PendingOperationDisabledReason string              `json:"pending_operation_disabled_reason"`
+	WebUrl                         string              `json:"web_url"`
```

Comment:
Is adding `WebUrl` here the right path forward? Using this as the link to open in the browser.